src/API/Site/Controllers/MerchantCenter/PriceBenchmarksController.php

```diff
@@ -69,10 +69,10 @@ protected function get_price_benchmarks_callback(): callable {
 				/** @var MerchantPriceBenchmarks $merchant */
 				$merchant = $this->container->get( MerchantPriceBenchmarks::class );
 
-				$benchmark_data = $merchant->get_benchmark_data( 'products', $this->prepare_query_arguments( $request ) );
+				$benchmark_data = $merchant->get_benchmark_data( $this->prepare_query_arguments( $request ) );
 				$benchmark_data = $this->prepare_item_for_response( $benchmark_data, $request );
 
-				$get_price_insights_product_view = $merchant->get_price_insights_product_view( 'products', $this->prepare_query_arguments( $request ) );
+				$get_price_insights_product_view = $merchant->get_price_insights_product_view( $this->prepare_query_arguments( $request ) );
 				$get_price_insights_product_view = $this->prepare_item_for_response( $get_price_insights_product_view, $request );
```

Comment:
We'll want to rename this as well

```suggestion
				$price_insights_data = $merchant->get_price_insights( $this->prepare_query_arguments( $request ) );
				$price_insights_data = $this->prepare_item_for_response( $get_price_insights_product_view, $request );
```